PackageExplorer/Utilities/NuGetExtensions.cs

```diff
@@ -0,0 +1,271 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGet.Frameworks
+{
+	/// <summary>
+	/// Class for Extensions related to NuGetFramework objects.
+	/// </summary>
+	public static class NuGetFrameworkExtensions
```

Comment:
Can you add some examples (`<example`) on this methods? I have difficulties to follow the logic here